compiler/rustc_middle/src/ty/visit.rs

```diff
@@ -82,9 +82,11 @@ pub trait TypeVisitable<'tcx>: fmt::Debug + Clone {
         self.has_vars_bound_at_or_above(ty::INNERMOST)
     }
 
-    #[instrument(level = "trace", ret)]
     fn has_type_flags(&self, flags: TypeFlags) -> bool {
-        self.visit_with(&mut HasTypeFlagsVisitor { flags }).break_value() == Some(FoundFlags)
+        let res =
+            self.visit_with(&mut HasTypeFlagsVisitor { flags }).break_value() == Some(FoundFlags);
+        trace!(?self, ?flags, ?res, "has_type_flags");
+        res
```

Comment:
why did you remove the `instrument` here?

Reply:
These flag computations were filling most of my logs and hiding the actually useful information. If I compress it a lot I can get it to 4 lines, but if I remove the `instrument` it can be a single line without new indentation